src/environments/environment.ts

```diff
@@ -4,5 +4,5 @@ const sentryDSN = 'https://xxxx@sentry.io/9999999';
 
 export const environment = {
     sentryDSN,
-    ...env
+    ...env.environment
```

Comment:
I did modify this line because when we use looks like this `environment.environment.production`. With that change we its come back to `environment.production`.

Reply:
Got it, might we need to check if we have `environment.environment....` in entire code to replace it just in case we have it.